.vscode/settings.json

```diff
@@ -1,3 +1,15 @@
 // Place your settings in this file to overwrite default and user settings.
```

Comment:
As soon as it's in gitignore, nobody will cloning the repo will get it. So I don't see the point of the point of the two-step process.